docs/inventories.rst

```diff
@@ -0,0 +1,801 @@
+  b. if schedules have not been set up, set them up following the prompts.
+
+12.  To configure notifications for the source, click the **Notifications** tab.
+
+  a. If notifications are already set up, use the toggles to enable or disable the notifications to use with your particular source.
+
+  b. If notifications have not been set up, set them up following the prompts.
+
+
+13. Review your entries and selections and click **Save** when done.
+
+Once a source is defined, it displays as an entry in the list of sources associated with the inventory. From the **Sources** tab you can perform a sync on a single source, or sync all of them at once. You can also perform additional actions such as scheduling a sync process, and edit or delete the source.
+
+
+.. _ug_inventory_sources:
+
+Inventory Sources
+^^^^^^^^^^^^^^^^^^^^^^
+
+To configure inventory sources, refer to `ug_inventories_plugins`.
+
+
+.. _ug_customscripts:
+
+Export old inventory scripts
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+.. index::
+    pair: inventories; custom script
+
+Despite the removal of the custom inventory scripts API, the scripts are still saved in the database. The commands described in this section allows you to recover the scripts in a format that is suitable for you to subsequently check into source control. Usage looks like this:
+
+
+::
+
+  $ awx-manage export_custom_scripts --filename=my_scripts.tar
+  Dump of old custom inventory scripts at my_scripts.tar
+
+Making use of the output:
+
+::
+
+  $ mkdir my_scripts
+  $ tar -xf my_scripts.tar -C my_scripts
+
+The naming of the scripts is ``_<pk>__<name>``. This is the naming scheme used for project folders.
+
+::
+
+
+  $ ls my_scripts
+  _10__inventory_script_rawhook             _19__                                       _30__inventory_script_listenhospital
+  _11__inventory_script_upperorder          _1__inventory_script_commercialinternet45   _4__inventory_script_whitestring
+  _12__inventory_script_eastplant           _22__inventory_script_pinexchange           _5__inventory_script_literaturepossession
+  _13__inventory_script_governmentculture   _23__inventory_script_brainluck             _6__inventory_script_opportunitytelephone
+  _14__inventory_script_bottomguess         _25__inventory_script_buyerleague           _7__inventory_script_letjury
+  _15__inventory_script_wallisland          _26__inventory_script_lifesport             _8__random_inventory_script_
+  _16__inventory_script_wallisland          _27__inventory_script_exchangesomewhere     _9__random_inventory_script_
+  _17__inventory_script_bidstory            _28__inventory_script_boxchild
+  _18__p                                    _29__inventory_script_wearstress
+
+
+Each file contains a script. Scripts can be ``bash/python/ruby/more``, so the extension is not included. They are all directly executable (assuming the scripts worked). If you execute the script, it dumps the inventory data.
+
+.. code-block:: bash
```

Comment:
The `console` lexer is supposed to be used for terminal sessions with prompts and outputs.